app/models/split_detail.rb

```diff
@@ -0,0 +1,58 @@
+class SplitDetail
+  include ActiveModel::Model
+
+  attr_accessor :split, :hypothesis, :assignment_criteria, :description, :owner, :location, :platform
```

Comment:
for all the attributes other than `:split`, can we use `delegate_attributes` for them?
If we do that, I believe we can remove all the getter methods in here and omit all the params in `split.update!`. 
We _may_ just need to make sure we grab the `split` param in the initialize before the other attributes tho:
```ruby
def initialize(params)
  raise ...
  self.split = params.delete(:split)
  super
end
```
I'm not 100% confident if this would be the right way of using `delegate_attributes`, so would like @samandmoore or anyone else to confirm